rules/windows/defense_evasion_posh_obfuscation_high_number_proportion.toml

```diff
@@ -119,12 +119,15 @@ from logs-windows.powershell_operational* metadata _id, _version, _index
     user.id
 
 // Filter for scripts with high numeric character ratio
-| where Esql.script_block_ratio > 0.30
+| where Esql.script_block_ratio > 0.35
 
 // Exclude Windows Defender Noisy Patterns
 | where not (
     file.directory == "C:\\ProgramData\\Microsoft\\Windows Defender Advanced Threat Protection\\Downloads" or
-    file.directory like "C:\\\\ProgramData\\\\Microsoft\\\\Windows Defender Advanced Threat Protection\\\\DataCollection*"
+    file.directory like (
```

Comment:
🚀

Reply:
We should double check that this is not minstacked to a specific version (e.g. 9.1.0). It's hard to tell in the [docs](https://www.elastic.co/docs/reference/query-languages/esql/functions-operators/operators#esql-like).